src/interfaces/store-interfaces/store-attribute-modify-interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface StoreAttributeModifyIF {
```

Comment:
Why not call it "StoreAttributeSetIF" ?

Reply:
I'll delete this file too.  It's not used at present.